phoenix-postgres-pgadmin/example_app/config/.secret.exs

```diff
@@ -0,0 +1,11 @@
+use Mix.Config
+
+config :example_app, ExampleApp.Repo,
+  username: "postgres",
+  password: "postgres",
+  database: "ExampleAppDB",
+  hostname: "localhost",
+  port: "5432",
+  virtual_host: "/",
+  show_sensitive_data_on_connection_error: true,
```

Comment:
I don't really know phoenix or elixir but this would be better if it was `false` by default no?

Reply:
You are right! Fixed!